gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -32,6 +32,19 @@ module.exports = function(env) {
     }
   }
 
+  if(env === 'development') {
```

Comment:
I think I want to refactor at some point to use `process.env` for this stuff instead.